src/config.ts

```diff
@@ -41,10 +41,10 @@ export function getSqlBasePath(config: Configuration): string {
  *
  * @returns {Promise<Configuration>}
  */
-export async function loadConfig(): Promise<Configuration> {
+export async function loadConfig(configFilename: string = CONFIG_FILENAME): Promise<Configuration> {
   log('Resolving config file.');
 
-  const filename = path.resolve(process.cwd(), CONFIG_FILENAME);
+  const filename = path.resolve(process.cwd(), configFilename);
```

Comment:
| Let's make sure all sync-db configurations are prefixed with `sync-db-*`.

https://github.com/leapfrogtechnology/sync-db/pull/119#issuecomment-807996558

Should we throw an exception if this configuration filename pattern doesn't match?

cc @mesaugat

Reply:
Yes we should. Please limit it to `sync-db.yml` and `sync-db-*.yml` file names.

Reply:
const filename = path.isAbsolute(configFilename)
    ? configFilename
    : path.join(process.cwd(), configFilename);